internal/client.go

```diff
@@ -315,18 +378,24 @@ func (client *Client) drawArchivedPlots() {
 	client.lastTable.ScrollToBeginning()
 }
 
-func (client *Client) selectActivePlot(row int, column int) {
-	s := ""
-	if log, found := client.activeLogs[row]; found {
-		for _, line := range log {
-			s += line
-		}
+func (client *Client) selectActivePlot(key string) {
+	client.logPlotId = key
+	client.lastTable.SetSelectedStyle(tcell.StyleDefault.Attributes(tcell.AttrReverse | tcell.AttrDim))
```

Comment:
I'm not sure how well this displays on linux, my terminal doesn't seem to be capturing the mouse.  Some tweaking on the attributes may be required.